pkgs/by-name/un/unhide-gui/package.nix

```
@@ -0,0 +1,61 @@
+{
+  fetchFromGitHub,
+  lib,
+  python3Packages,
+  python3,
+  unhide,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "unhide";
```

```suggestion
  pname = "unhide-gui";
```
It should match top-level.